project.clj

```diff
@@ -1,14 +1,14 @@
-(defproject clojure-elastic-apm "0.1.1"
```

Comment:
We should consider [creating new releases on GH](https://help.github.com/en/articles/creating-releases) when we bump the version number.